### ISSUE TITLE
Email notifier is not saving emails to db

### ISSUE THREAD

**Issue by nobodyspecial** [pymedusa/Medusa#7459]
**Describe the bug**
Show notification list in email notifier is not saving emails to database.

**To Reproduce**
1. Go to Notifications -> Social -> Email
2. Click on Select a show -> Any show -> Add email -> Save
3. Email appears to save
4. Emails not sent after snatch or post-process

**Expected behavior**
Emails should save to db

**Screenshots**
![medusa](https://user-images.githubusercontent.com/652026/70375687-5a8cf580-18c6-11ea-871a-6918240fcf55.png)

**Medusa (please complete the following information):**
 - OS: Windows-2003Server-5.2.3790-SP2
 - Branch: master, develop
 - Commit: fcb5a1969dcd4ea0b6fb4a9debbf60179b0bbf4b
 - Python version: 2.7.10
 - Database version: 44.14

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
<details>

```
2019-12-07 07:47:18 DEBUG	APIv2-Thread-1 :: Broken providers already updated in the last hour
```
</details>

**Additional context**

When saving an email, the above message appears in the debug log. I have checked the main.db file and confirmed notify_list is empty for all shows in the tv_shows table after saving. Also, the highlighted text in the image is wrong but unrelated to this issue I think.

**Comment by medariox** [comment]
You need to click on the `+` and save afterwards.

**Comment by nobodyspecial** [comment]
> 
> 
> You need to click on the `+` and save afterwards.

Yes, I know this. It is only saving on the frontend and not in db.

**Comment by medariox** [comment]
What do you mean by that? The settings are saved in `config.ini`, not in the DB.

**Comment by nobodyspecial** [comment]
> 
> 
> What do you mean by that? The settings are saved in `config.ini`, not in the DB.

Are you sure the individual show notification emails are saved in config.ini? I know the rest of the email settings are saved there and those work fine.

**Comment by p0psicles** [comment]
I believe those are saved in a table. But I have to double check

**Comment by nobodyspecial** [comment]
notify_list in tv_shows table

**Comment by p0psicles** [comment]
Yes. Like prowl and email is stored there

**Comment by nobodyspecial** [comment]
Can anyone confirm this bug or is there something wrong with my install?

**Comment by chaddoncooper** [comment]
I am also seeing the issue, the individual show's email addresses are being sent correctly in a separate payload but they aren't persisted to the db. I am clicking the plus after adding the show.
Another area using a similar mechanism - required words saves to the is_required_word field just fine.

**Comment by nobodyspecial** [comment]
What more info is needed on this? I'm now running this in a docker in unraid and it's still not fixed. In fact now the "Select a show" drop down is missing.

**Comment by p0psicles** [comment]
Fixed in #8535
